.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+# Runs CI tasks on pull requests and pushes
+
+name: Continuous Integration
+
+on:
+  pull_request:
+    branches:
+      - dev
+      - main
+  push:
+    branches-ignore:
+      - main
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+
+      - name: Cache Node.js modules
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.OS }}-node-
+            ${{ runner.OS }}-
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: Run linter
+        run: npm run lint
+
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+
+      - name: Cache Node.js modules
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.OS }}-node-
+            ${{ runner.OS }}-
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: Run Tests
+        run: npm run test
+
+      - name: Publish Coverage Report
```

Comment:
Scrap this, remove it from package.json, use https://github.com/marketplace/actions/coveralls-github-action instead. Having this command is not useful offline.